app/components/action_button.rb

```diff
@@ -1,14 +1,14 @@
 # frozen_string_literal: true
 
-# Draws a blue button in the side menu
+# Draws a red button on the item detail page
 class ActionButton < ApplicationComponent
   # @param [Hash] properties the button properties
-  def initialize(label:, url:, method: nil, confirm: nil, new_page: nil, check_url: nil, disabled: nil)
+  def initialize(label:, url:, method: nil, confirm: nil, open_modal: false, check_url: nil, disabled: nil)
     @label = label
```

Comment:
I got tripped up by this "new_page" attribute, which basically means "don't create a modal window" when clicked.  It you don't add a "new_page", then anything other than a GET method will not work, which I found confusing.  I switched the parameter to "open_modal" instead and adjusted the buttons to reflect this.